src/AirshipBindings.NETStandard/AirshipBindings.NETStandard.Android/MessagePageRenderer.cs

```diff
@@ -0,0 +1,166 @@
+﻿using Android.Content;
+using Android.Runtime;
+using Android.Views;
+using Android.Webkit;
+using Android.Widget;
+using UrbanAirship.NETStandard.MessageCenter;
+using UrbanAirship.RichPush;
+using UrbanAirship.Widget;
+using Xamarin.Forms.Platform.Android;
+
+namespace UrbanAirship.NETStandard.Android
+{
+    public class MessagePageRenderer : PageRenderer, RichPushInbox.IFetchMessagesCallback
+    {
+        private class MessageWebViewClient : UAWebViewClient
+        {
+            private MessagePageRenderer renderer;
```

Comment:
This might seem circular, but it's the only way to get access to variables in the outer scope, since C# doesn't have an equivalent of anonymous inner classes as in Java